audio.py

```diff
@@ -88,3 +137,20 @@ def compute_duration(audio_path, backend = 'ffmpeg'):
 	cmd = ['ffprobe', '-v', 'error', '-show_entries', 'format=duration', '-of', 'default=noprint_wrappers=1:nokey=1'
 			] if backend == 'ffmpeg' else ['soxi', '-D'] if backend == 'sox' else None
 	return float(subprocess.check_output(cmd + [audio_path]))
+
+
+if __name__ == '__main__':
+	parser = argparse.ArgumentParser()
+	parser.add_argument('--audio_path', type = str, required = True)
```

Comment:
Add a script for generating test audio files: in wav (s16le), mp3, opus, gsm (to be saved into `./data`). There is an example in https://github.com/vadimkantorov/readaudio/blob/master/README.md#usage

Please report the perf measures in README.

Reply:
Please name the "audio_path" argument consistently with the rest of the codebase: `--audio-path`

Reply:
done